index.js

```diff
@@ -99,6 +100,10 @@ alfy.config = new Conf({
 	cwd: alfy.alfred.data
 });
 
+alfy.userConfig = new AlfredConfig({
+	cwd: alfy.alfred.data
```

Comment:
This is the default. I wasn't sure if we should pass it through but might be good to be explicit? Could keep it with just `new AlfredConfig()` though.

Reply:
If it's the default in `AlfredConfig`, I don't really see the need to specify it here.